packages/orchestration/src/typeGuards.js

```diff
@@ -160,8 +160,11 @@ export const chainFacadeMethods = harden({
 /**
  * for google/protobuf/timestamp.proto, not to be confused with TimestampShape
  * from `@agoric/time`
+ *
+ * `seconds` should be a BigInt but since it goes through JSON it is encoded as
+ * a string
  */
-export const TimestampProtoShape = { seconds: M.nat(), nanos: M.number() };
+export const TimestampProtoShape = { seconds: M.string(), nanos: M.number() };
```

Comment:
Good catch. I would like us to add an E2E test for the LocalOrchAcct staking flows (`Undelegate` for this particular change) , perhaps in the course of https://github.com/Agoric/agoric-sdk/issues/10048 and/or https://github.com/Agoric/agoric-sdk/issues/9838

Reply:
fwiw I didn't catch this analytically. it was caught by your improvement to JsonSafe in telescope that we finally have in agoric-sdk.

That required this change:
```diff
      return /** @type {JsonSafe<MsgUndelegateResponse>} */ ({
        // 5 seconds from unix epoch
        completionTime: { seconds: 5n, nanos: 0 },
        completionTime: { seconds: '5', nanos: 0 },
```

Which then required the guard change.